tests/util.py

```diff
@@ -30,6 +28,7 @@
 )
 
 MODEL_DIR = Path("./tests/models/")
+ZIP_DIR = Path("./tests/zips/")
```

Comment:
This folder either to be git-ignored.

Reply:
Done! I meant to do that originally, but got distracted. 😂